src/libs/kata-sys-util/src/protection.rs

```diff
@@ -72,42 +70,20 @@ pub fn available_guest_protection() -> Result<GuestProtection, ProtectionError>
 
     arch_guest_protection(
         TDX_SYS_FIRMWARE_DIR,
-        TDX_CPU_FLAG,
         SEV_KVM_PARAMETER_PATH,
         SNP_KVM_PARAMETER_PATH,
     )
 }
 
-#[cfg(target_arch = "x86_64")]
-fn retrieve_cpu_flags() -> Result<String> {
-    let cpu_info =
-        crate::cpu::get_single_cpu_info(crate::cpu::PROC_CPUINFO, crate::cpu::CPUINFO_DELIMITER)?;
-
-    let cpu_flags =
-        crate::cpu::get_cpu_flags(&cpu_info, crate::cpu::CPUINFO_FLAGS_TAG).map_err(|e| {
-            anyhow!(
```

Comment:
As you removed this one, the static checks are complaining about anyhow imported but not used.